pom.xml

```diff
@@ -440,6 +446,233 @@
+                <visitors>UnreadFields,FindRefComparison,FindOpenStream,InvalidJUnitTest,FindDeadLocalStores,UnreadFields,BooleanReturnNull</visitors>
+                <!-- <timeout>60000</timeout> -->
+                <plugins>
+                  <plugin>
+                    <groupId>org.eclipse.smarthome.build-tools</groupId>
+                    <artifactId>static-code-analysis-config</artifactId>
+                    <version>${project.version}</version>
+                  </plugin>
+                </plugins>
+              </configuration>
+              <executions>
+                <execution>
+                  <phase>${phase.prop}</phase>
+                  <goals>
+                    <goal>check</goal>
+                  </goals>
+                </execution>
+              </executions>
+              <dependencies>
+                <dependency>
+                  <groupId>com.google.code.findbugs</groupId>
+                  <artifactId>findbugs</artifactId>
+                  <version>${findbugs.version}</version>
+                </dependency>
+                <dependency>
+                  <groupId>org.eclipse.smarthome.build-tools</groupId>
+                  <artifactId>static-code-analysis-config</artifactId>
+                  <version>${project.version}</version>
+                </dependency>
+              </dependencies>
+            </plugin>
+            <plugin>
+              <groupId>org.codehaus.mojo</groupId>
+              <artifactId>exec-maven-plugin</artifactId>
+              <version>${maven.exec.plugin.version}</version>
+              <executions>
+                <execution>
+                  <phase>${phase.prop}</phase>
+                  <goals>
+                    <goal>java</goal>
+                  </goals>
+                </execution>
+              </executions>
+              <configuration>
+                <arguments>
+                  <argument>${report.directory}</argument>
+                </arguments>
+                <includeProjectDependencies>false</includeProjectDependencies>
+                <includePluginDependencies>true</includePluginDependencies>
+                <executableDependency>
+                  <groupId>org.eclipse.smarthome.build-tools</groupId>
+                  <artifactId>static-code-analysis-report</artifactId>
+                </executableDependency>
+                <mainClass>org.eclipse.smarthome.buildtools.report.ScanReportUtility</mainClass>
+              </configuration>
+              <dependencies>
+                <dependency>
+                  <groupId>org.eclipse.smarthome.build-tools</groupId>
+                  <artifactId>static-code-analysis-report</artifactId>
+                  <version>${project.version}</version>
+                  <type>jar</type>
+                </dependency>
+              </dependencies>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+      </build>
+    </profile>
+    <profile>
```

Comment:
This may seem a little confusing ...
I use a separate profile to activate the plugins for FindBugs, PMD and Checkstyle and to apply specific configuration data for a set of plug-ins (in this case bundles).